### PR TITLE
adding pr_check and build_deploy

### PR DESCRIPTION
**Pull request by gburges**

### CODE REVIEW

File: .rhcicd/build_deploy.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# --------------------------------------------
+# Export vars for helper scripts to use
+# --------------------------------------------
+# name of app-sre "application" folder this component lives in; needs to match for quay
+export COMPONENT="notifications"
+export WORKSPACE=${WORKSPACE:-$APP_ROOT} # if running in jenkins, use the build's workspace
+export APP_ROOT=$(pwd)
+export NODE_BUILD_VERSION=12
```
Comment (josejulio) [inline .rhcicd/build_deploy.sh]: I don't think we really have an specific version, but could we use the 16?
```suggestion
export NODE_BUILD_VERSION=16
```

File: .rhcicd/pr_check.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# --------------------------------------------
+# Export vars for helper scripts to use
+# --------------------------------------------
+# name of app-sre "application" folder this component lives in; needs to match for quay
+export COMPONENT="notifications"
+export WORKSPACE=${WORKSPACE:-$APP_ROOT} # if running in jenkins, use the build's workspace
+export APP_ROOT=$(pwd)
+export NODE_BUILD_VERSION=12
```
Comment (josejulio) [inline .rhcicd/pr_check.sh]: same as above
```suggestion
export NODE_BUILD_VERSION=16
```

**Review by josejulio — APPROVED** [review]

File: .rhcicd/build_deploy.sh [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+source <(curl -sSL $COMMON_BUILDER/src/frontend-build.sh)
+BUILD_RESULTS=$?
+
+# Stubbed out for now
+mkdir -p $WORKSPACE/artifacts
+cat << EOF > $WORKSPACE/artifacts/junit-dummy.xml
+<testsuite tests="1">
+    <testcase classname="dummy" name="dummytest"/>
+</testsuite>
+EOF
```
Comment (gwenneg) [inline .rhcicd/build_deploy.sh:31]: Is this needed for the frontend?

File: .rhcicd/pr_check.sh [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+source <(curl -sSL $COMMON_BUILDER/src/frontend-build.sh)
+BUILD_RESULTS=$?
+
+# Stubbed out for now
+mkdir -p $WORKSPACE/artifacts
+cat << EOF > $WORKSPACE/artifacts/junit-dummy.xml
+<testsuite tests="1">
+    <testcase classname="dummy" name="dummytest"/>
+</testsuite>
+EOF
```
Comment (gwenneg) [inline .rhcicd/pr_check.sh:31]: Same question.

File: .rhcicd/build_deploy.sh (hunk 3 above)
Comment (josejulio) [reply .rhcicd/build_deploy.sh:31]: Yes, as of right now this is required. It might be removed/changed in the future

Comment (gburges) [reply .rhcicd/build_deploy.sh:31]: yes, it is needed, because if the xml doesn't exist, the jenkins job fails

File: .rhcicd/pr_check.sh (hunk 4 above)
Comment (gburges) [reply .rhcicd/pr_check.sh:31]: Same, it's needed for the jenkins job to succeed

File: .rhcicd/build_deploy.sh (hunk 3 above)
Comment (gwenneg) [reply .rhcicd/build_deploy.sh:31]: Ok I thought this was only needed when JUnit was involved in the build.

**Review by BlakeHolifield — APPROVED** [review]

**Pull request merged**